src/client/components/TimeZone/TimeZone.component.js

```diff
@@ -1,33 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import DropDown from '../Dropdown/DropDown';
-import PropTypes, { shape } from 'prop-types';
 import 'moment-timezone';
 
-export const TimeZoneDropDown = (props) => {
+const timeZones = [
+  {
+    id: 1,
+    name: 'Europe/Copenhagen',
+  },
+  {
+    id: 2,
+    name: 'Europe/Dublin',
+  },
+  {
+    id: 3,
+    name: 'Europe/Budapest',
+  },
+  {
+    id: 4,
+    name: 'Australia/Sydney',
+  },
+];
+
+export const TimeZoneDropDown = () => {
+  const [name, setName] = useState('');
```

Comment:
I only meant to move the timeZones array inside. If the component doesn't expose `name` and `setName`, there is no way to figure out what timeZone was picked in the owner page.